### PR TITLE
[Fortune] Add endpoint for workers stats 

### LINKED ISSUE #1710: Fortune Exchange Oracle - Get assignment stats endpoint
**Description**
Implement an endpoint that follows the [standard](https://docs.humanprotocol.org/human-tech-docs/exchange-oracle#get-assignment-stats). 

**Motivation**
Implement standard in Fortune Exchange Oracle

**Implementation details** 
It should return an object as the one in the docs.

### PR DESCRIPTION
**Pull request by mrhouzlane**
## Description

Implement an endpoint that follows the [standard](https://docs.humanprotocol.org/human-tech-docs/exchange-oracle#get-assignment-stats).

## Summary of changes

- Add `StatsController` 
- Add `StatsService` 
- Add `StatsDto` with a constructor. 
- Add `StatsModule`

Closes #1710

### CODE REVIEW

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.dto.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export class AssignmentStatsDto {
+  assignments_total: number;
+  submissions_sent: number;
+  assignments_completed: number;
+  assignments_rejected: number;
+  assignments_expired: number;
```
Comment (flopez7) [inline packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.dto.ts:6]: We have an interceptor inside API to convert camelCase objects to snake_case so let's use camelCase inside the code. Also, I think this should be inside `stats.dto.ts`

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.spec.ts
Comment (flopez7) [inline packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.spec.ts:1]: Remove this file

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.controller.ts
Comment (flopez7) [inline packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.controller.ts:1]: This should be a new Stats controller

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+      return acc;
+    }, initialStats);
+
+    const finalStats = new AssignmentStatsDto();
+    finalStats.assignments_total = stats.assignments_total;
+    finalStats.submissions_sent = stats.submissions_sent;
+    finalStats.assignments_completed = stats.assignments_completed;
+    finalStats.assignments_rejected = stats.assignments_rejected;
+    finalStats.assignments_expired = stats.assignments_expired;
```
Comment (flopez7) [inline packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.ts:54]: Why don't you use the constructor?

**Review by flopez7 — CHANGES_REQUESTED** [review]

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.spec.ts
Comment (mrhouzlane) [reply packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.spec.ts:1]: Removed

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.controller.ts
Comment (mrhouzlane) [reply packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.controller.ts:1]: Changed to `StatsController`

File: packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.ts (hunk 2 above)
Comment (mrhouzlane) [reply packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.service.ts:54]: Added a constructor

**Review by flopez7 — CHANGES_REQUESTED** [review]
As it's a new module, you have to create the new `stats.module.ts` and also import the `StatsModule` inside `app.module.ts`

**Pull request merged**